src/less/components/footer.less

```diff
@@ -1,10 +1,20 @@
-footer {
-    &.page-footer {
-        padding: 1rem 2rem;
-        border-top: 8px solid @brand-primary;
+.page-footer {
+    padding: 1rem 2rem;
+    border-top: 8px solid @brand-primary;
 
-        > p {
-            font-family: @font-swedbank-headline;
+    > p {
+        font-family: @font-swedbank-headline;
+    }
```

Comment:
Either add this to the footer or to the .page-footer-rights class?
.page-footer-rights class in my opinion.